task-list/app/views/tasks/index.html.erb

```diff
@@ -22,6 +22,11 @@
       <td id="task_link_cell">
```

Comment:
I don't think it is a good idea to use the `id` attribute here and in the following `td` tags. Because this code is within an `each` loop from line 13, it will be repeated for every element in `@tasks`, but we should only have a single tag with a given `id` value.

I'd recommend either changing these attributes to `class`, or to use ERB tags to generate a unique ID like: `<td id="task_link_cell_<%= task.id %>">`.